packages/serverless/test/integrations/aws_lambda.test.ts

```diff
@@ -0,0 +1,753 @@
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const childProcess = require('child_process');
+const uuid = require('uuid');
+const lambda = new AWS.Lambda({ apiVersion: '2015-03-31', region: 'us-east-2' });
+
+function runLambdaFunction(additionalLambdaPrelude: any, callback: any) {
+  if (
+    process.env.AWS_ACCESS_KEY_ID == undefined &&
+    process.env.AWS_SECRET_ACCESS_KEY == undefined &&
+    process.env.AWS_LAMBDA_ROLE == undefined
+  ) {
+    throw new Error('AWS credentials or roles are not set correctly');
+  }
+
+  const lambdaPrelude = `
+const Sentry = require("@sentry/serverless");
+const http = require("http");
+const AwsLambda = Sentry.Integrations.AWSLambda;
+const { HTTPSTransport } = require("@sentry/node/dist/transports");
+
+class testTransport extends HTTPSTransport {
+  constructor() {
+    super(...arguments);
+  }
+  async sendEvent(event) {
+    console.log("Event:", JSON.stringify(event));
+  }
+}
+exports.handler = ${additionalLambdaPrelude.scenario} (event, context) => {
+  // Configure the Sentry SDK.
+  Sentry.init({
+    dsn:
+      "https://e16b3f19d01b4989b118f20dbcc11f87@o388065.ingest.sentry.io/5224330",
+    transport: testTransport,
+    integrations: [new AwsLambda({ context, timeoutWarning: ${additionalLambdaPrelude.timeoutWarning}})],
+  });
+
+  ${additionalLambdaPrelude.error}
+
+}; `;
+
+  const packageDir: string = './test/testCase';
+  const setupJsonFile: string = './test/setup.json';
+  const buildDist: string = './dist';
+
+  if (!fs.existsSync(buildDist)) {
+    throw Error('Build not found in specified directory.');
+  } else {
+    childProcess.execSync('cp -r dist esm ./test/serverless/');
+  }
+
+  if (fs.existsSync(packageDir)) {
+    childProcess.execSync('cd ./test/; rm -rf testCase; cd ..; cd ..;');
+  }
+
+  fs.mkdirSync(packageDir);
+  fs.appendFile(`${packageDir}/index.js`, lambdaPrelude, (err: any) => {
```

Comment:
It's the test, we can use `sync` methods everywhere here. There's no point in using async-callback methods here.

Reply:
Instead of using a prelude, you can create a test fixture file instead. It'll be easier to work with.